go/kbnm/main.go

```diff
@@ -86,6 +86,11 @@ func main() {
 		out = nativemessaging.NewNativeJSONEncoder(os.Stdout)
 	}
 
+	if *versionFlag {
```

Comment:
This is going to conflict with https://github.com/keybase/client/pull/6524 which was merged, FYI. Ideally keep my version, or use Println/add a newline.